src/AlphaCPU.cpp

```diff
@@ -392,7 +392,7 @@ CAlphaCPU::CAlphaCPU(CConfigurator *cfg, CSystem *system)
  * Initialize the CPU.
  **/
 void CAlphaCPU::init() {
-  memset(&state, 0, sizeof(state));
+  state = {};
```

Comment:
This change was necessary because the state now contains an std::list<> that must be properly constructed. Doing a memset() to zero out the memory leaves the std::list<> in an invalid state which crashes the process. The Modern C++ syntax with which I've replaced the memset() will zero-initialise simple data types but default construct the std::list<>.

Reply:
es40 was written in "C with classes" style (and also before C++11), so C-style code is common here. I want to eventually rewrite everything as idiomatic C++, but it will take some time and patience.